packages/idyll-document/src/index.js

```diff
@@ -77,6 +78,7 @@ class IdyllDocument extends React.Component {
           }}
           initialState={this.props.initialState || (this.idyllContext ? this.idyllContext.data() : {})}
           ast={ this.props.ast || this.state.ast }
+          userViewComponent={UserViewPlaceholder}
```

Comment:
I think we just want to pass down `this.props.userViewComponent` here, otherwise it would always show the placeholder.